device.yml

```
@@ -84,6 +84,71 @@ registers:
     access: Event
     type: S16
     length: 6
+    description: Flow meters values.
+    payloadSpec:
+      Flow0X:
+        offset: 0
+        description: Flow meter value on port 0 axis X.
+      Flow0Y:
+        offset: 0
+        description: Flow meter value on port 0 axis Y.
+      Flow0Squal:
+        offset: 0
+        description: Signal quality of flow meter 0.
+      Flow1X:
+        offset: 0
+        description: Flow meter value on port 0 axis X.
+      Flow1Y:
+        offset: 0
+        description: Flow meter value on port 0 axis Y.
+      Flow1Squaç:
+        offset: 0
+        description: Signal quality of flow meter 1.
+  ProductIdPort0:
+    address: 47
+    access: Read
+    type: U8
+    description: Contains de flow meter ID connected to Port 0.
+  ProductIdPort1:
+    address: 48
+    access: Read
+    type: U8
+    description: Contains de flow meter ID connected to Port 1.
```

```suggestion
  OpticalFlowPid0:
    address: 47
    access: Read
    type: U8
    description: The product ID of the optical flow sensor connected to Port 0.
  OpticalFlowPid1:
    address: 48
    access: Read
    type: U8
    description: The product ID of the optical flow sensor connected to Port 1.
```

I have proposed to rename this register to make it clear we are talking here about "optical flow sensor" product IDs.